lib/bin/supllib/readme.rst

```diff
@@ -0,0 +1,25 @@
+SUPL Client library
+###################
+
+This is the SUPL Client library module that integrates SUPL client library to NCS. The
+GPS sample located at nrf/samples/nrf9160/gps has support for receiving AGSP
+data over the SUPL protocol if enabled and the library is installed correctly.
+
+Downloading
+***********
+The SUPL client library zip file must be downloaded from [SOME PLACE ON NORDICSEMI.COM]
+
+Installing
+**********
+Unzip the downloaded zip file to nrf/ext/bin/supl/ while keeping the folder structure
```

Comment:
The CMakeLists.txt states it's `ext/supl`, not `ext/bin/supl`.

Reply:
ok